courses/udacity_intro_to_tensorflow_for_deep_learning/l07c01_saving_and_loading_models.ipynb

```diff
@@ -1,5 +1,33 @@
 {
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "accelerator": "GPU",
+    "colab": {
+      "name": "l07c01_saving_and_loading_models.ipynb",
+      "provenance": [],
+      "private_outputs": true,
+      "collapsed_sections": [],
+      "toc_visible": true,
+      "machine_shape": "hm",
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "display_name": "Python 3",
+      "name": "python3"
+    }
+  },
   "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
```

Comment:
Please remove this cell

Reply:
@lamberta I've removed this cell, pls see if its correct. Also, I tried to find this cell in my fork but I couldn't find it, I don't know why. So I changed it here itself. Also, I'm seeing changes I didn't do in the diff. Do u know why is it so??
Thanks a lot for help!

Reply:
I think this cell is added when we check the "Include Link to Colab" option while saving the colab to Github